app-emulation/aranym/aranym-git.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Atari Running on Any Machine"
+DESCRIPTION="
+ARAnyM (Atari Running on Any Machine) is a multiplatform virtual machine for \
+running Atari ST/TT/Falcon operating systems and applications.
+"
+HOMEPAGE="http://github.com/aranym/aranym"
+COPYRIGHT="2001-2018 ARAnyM developer team"
+LICENSE="GNU GPL v2"
+
+REVISION="1"
+SOURCE_URI="git://github.com/aranym/aranym#HEAD"
+SOURCE_DIR="aranym-master"
```

Comment:
```diff
 LICENSE="GNU GPL v2"
-
 REVISION="1"
+srcGitRev="0dbddebf0bbdaced8aeb21b62ff190b71e14b39a"
-SOURCE_URI="git://github.com/aranym/aranym#HEAD"
+SOURCE_URI="https://github.com/aranym/aranym/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="4082b703d558e4745c2cb93a5da9c4371cf3fbad924ddca9feaa5fc9ef5e50c6"
-SOURCE_FILENAME="aranym-$gitCommit.tar.gz"
+SOURCE_FILENAME="aranym-$portVersion-$srcGitRev.tar.gz"
-SOURCE_DIR="aranym-master"
+SOURCE_DIR="aranym-$srcGitRev"
```
We prefer to avoid `#HEAD` and use a specific git revision, so that we can include the checksum.

BTW, the recipe should be renamed to `aranym-1.0.3~git.recipe`.

Reply:
> We prefer to avoid #HEAD and use a specific git revision,

I understand that, but doesn't that mean you have to change the recipe every time a new commit is made ?

Reply:
Exact.
Well, after all, I think it would be OK to keep the `#HEAD` (and no checksum) if you replace `x86_64` by `?x86_64` in `ARCHITECTURES`.